pom.xml

```
@@ -136,6 +136,7 @@
         <maven.sources.version>3.0.1</maven.sources.version>
         <maven.surefire.version>2.18.1</maven.surefire.version>
         <maven.war.version>2.2</maven.war.version>
+        <gitflow.maven.plugin.version>1.16.0</gitflow.maven.plugin.version>
```

```suggestion
        <maven.gitflow,plugin.version>1.16.0</maven.gitflow.plugin.version>
```
And they are actually alphabetically ordered. 

Bonus point if you change `<dockerfile.maven.version>` too. 👼

---

It's actually correct by the plugin name.

---

If I am offer an alternative, I have committed a change in the following fashion:
The version property names 
 - now follow the logic of `plugin-artifact-name` -> `plugin.artifact.name.version`
 - are now alphabetically ordered
 - and the unused properties are removed (`maven.compiler.source`, `maven.compiler.target`)

How about that, @benedeki?